CHANGELOG.mkd

```diff
@@ -2,6 +2,8 @@ CHANGELOG
 =========
 
 Unreleased
+
+- Added an interface to R10K::Source::Base named `reload!` for updating the environments list for a given deployment; `reload!` is called before deployment purges. [#1172](https://github.com/puppetlabs/r10k/pull/1172) 
```

Comment:
I would add that this fixes a race condition when running r10k in parallel, so we remember what prompted this.